Cargo.toml

```diff
@@ -10,6 +10,9 @@ members = [
     "plugins/c8y_log_plugin",
     "plugins/c8y_remote_access_plugin",
     "plugins/tedge_apt_plugin",
+    "plugins/tedge_configuration_plugin",
+    "plugins/tedge_configuration_plugin",
+    "plugins/tedge_configuration_plugin",
```

Comment:
Why 3 times?

Reply:
Genuinely, I have no idea what happened here but when I tried to remove additional two positions it was coming back to this state. Reinstall of vscode extension helped but that was some weird bug.